doc/quickstart/liblas_quickstart.rst

```diff
@@ -24,7 +28,7 @@ you might want to do on LAS data, and the utilities and approaches to
 take to complete those tasks. The demo data can be found in :file:`/home/user/data/las/srs.las`
```

Comment:
Yes, this demo data srs.las file exists but I couldn't follow the procedure.

Reply:
Perhaps this is a file permissions issue.